packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,162 @@
+    nomad_ca_cert = ""
+
+    # The field should contain the base64 encoded value of the Nomad cert. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CLIENT_CERT environment variable.
+    nomad_cert = ""
+
+    # The field should contain the base64 encoded value of the Nomad cert's key. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CLIENT_KEY environment variable.
+    nomad_cert_key = ""
+
+    # TFE Redis password. Mapped to the TFE_REDIS_PASSWORD environment variable.
+    redis_password = ""
+
+    # TFE Vault encryption key. Mapped to the TFE_ENCRYPTION_PASSWORD environment variable.
+    tfe_encryption_password = ""
+
+    # Password for the registry where the TFE image is hosted. Mapped to the TFE_IMAGE_REGISTRY_PASSWORD environment variable.
+    tfe_image_registry_password = ""
+
+  }
+  ```
+  
+  Update the `path` variable if default value of `job_name` is overridden in the `var.hcl` file.
+  The variables can be created as below by passing the `spec.nv.hcl` file we create above:
+
+  ```bash
+  $ nomad var put @spec.nv.hcl
+  ```
+
+
+<!-- Include information about how to use your pack -->
+
+# Pack Information
+
+After completing prerequisites, the pack can be run using the following command:
+```bash
+$nomad-pack run tfe_fdo_nomad -f var.hcl
+```
+
+The `var.hcl` file should contain the necessary variables required for the pack to run. The variables are listed below.
+
+## Variables
+
+These variables may be set to change the behavior of the TFE. Note that some of these variables come with default configuration while the rest need to provided for the pack deployment to succeed.
+<!-- Include information on the variables from your pack -->
+
+## Configuration
+
+| Name                                         | Required | Default                                      | Comments                                                                                                                                                                                              |
+|----------------------------------------------|----------|----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `job_name`                                   | no       | `"tfe-job"`                                  | Override the Nomad job name.                                                                                                                                                                          |
+| `datacenters`                                | no       | `"dc1"`                                      | Nomad datacenters where the task in the jobs will be spread.                                                                                                                                          |
```

Comment:
Is there any reason not to have default value be the default value for Nomad (i.e. empty which is the same as `"*"`)?